rules-reviewed/eap7/eap6/hsearch.windup.xml

```diff
@@ -330,6 +330,8 @@
         <rule id="hsearch-00110">
             <when>
                 <javaclass references="org.hibernate.search.backend.configuration.impl.IndexWriterSetting.{constant}">
+                    <location>ENUM_CONSTANT</location>
+                    <location>IMPORT</location>
```

Comment:
it just searches all type reference locations. see http://windup.github.io/windup/docs/latest/javadoc/org/jboss/windup/ast/java/data/TypeReferenceLocation.html